docs/team/mep.md

```diff
@@ -0,0 +1,265 @@
+MMTk Enhancement Proposal (MEP)
+===============================
+
+An MMTk Enhancement Proposal (MEP) is a more formal variant of issue.  It has a special format, and
+will undergo a more thorough review process.  Its goal is helping the MMTk developers making more
+informed decisions.
+
+MEP is inspired by the Java Enhancement Proposal, described in https://openjdk.org/jeps/1  However,
+unlike JEP which is more open-ended, MEP is more focused on making design decisions.
+
+# When is MEP required?
+
+An MEP is required when making a significant change to the design of the MMTk core.  It is
+applicable to any kind of significant changes, including but not limited to:
+
+-   Bug fixes, including performance bug fixes, that require change to a major part of the MMTk
+    core.
+-   Changes to the MMTk core to implement a feature demanded by bindings.
+-   Major refactoring to the MMTk core.
+
+One purpose of MEP is reducing the risks to the future development of MMTk.  Large-scale changes and
+public API changes usually indicate such risks, but these are only indicators, not criteria.  The
+assessment of risks is subjective, and we need to discuss in order to reach consensus.
+
+Note: JEP is also required for things that "require two or more weeks of engineering effort" and/or
+"are in high demand by developers or customers".  We don't judge whether we need MEP based on
+engineering effort or public demand.  Many PRs for MMTk require multiple weeks of work and rigorous
+testing, but most of them can be settled with regular issues and PRs.  We label priorities using
+GitHub issue tags, such as `P-normal`, `P-high`, etc.  If a feature is requested often, and we have
+man power for that, we can raise the priority.
+
+# Format
+
+A MEP will be posted as a GitHub issue in the `mmtk-core` repository.  It should contain certain
```

Comment:
The only thing that is required is the `MEP` tag. The others are just our normal convention about Github issues -- we don't need to discuss too much here. Like the `A-` tags and `C-` tags, we are not requiring the tags for issues (many issues are not tagged with those). `G-` tags only include `G-safety` and `G-performance`, which is clearly not covering all the issues. Thus `G-` tags should not be required as well.

Reply:
We may want a separate doc about our policies about Github issues. Some of the discussions about tags should go into that doc (which does not need to be a part of this PR and this doc).

Reply:
I removed the sentences that mentiond the `{A,C,G}-*` tags, and only kept the `MEP` tag.